.github/workflows/delayed-pipelines.yml

```
@@ -0,0 +1,307 @@
+        name: Calculate Version
+        uses: codebeltnet/minver-calculate@v2
+
+      - name: Download cuemon.snk file
+        uses: codebeltnet/gcp-download-file@v1
+        with:
+          serviceAccountKey: ${{ secrets.GCP_TOKEN }}
+          bucketName: ${{ secrets.GCP_BUCKETNAME }}
+          objectName: cuemon.snk
+
+      - name: Set environment variable for projects
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:81: Double quote to prevent globbing and word splitting

(shellcheck)